### ISSUE TITLE
Can't use Ctrl + V  paste  picture?

### ISSUE THREAD

**Issue by yeyonghao1** [wekan/wekan#2589]
![image](https://user-images.githubusercontent.com/22569933/62764007-766ebf00-babf-11e9-838b-b36ac9f2ee44.png)

**Comment by xet7** [comment]
@whowillcare 

Pasting image gives error `summernote is not a function` in browser console. Can you fix this?

Complete errors:
```
4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:77 WebSocket connection to 'ws://localhost/sockjs/308/n7f79y9u/websocket' failed: Error in connection establishment: net::ERR_BLOCKED_BY_CLIENT
O.websocket @ 4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:77
4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:1 Exception from Tracker afterFlush function: undefined
4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:1 TypeError: $(...).summernote is not a function
    at HTMLTextAreaElement.<anonymous> (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:323:333039)
    at Function.each (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:19:14472)
    at ie.fn.init.each (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:19:12315)
    at f.TemplateInstance.<anonymous> (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:323:333010)
    at http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:34669
    at Function.e._withTemplateInstanceFunc (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:37698)
    at n (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:34575)
    at f.View.<anonymous> (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:35540)
    at http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:18157
    at Object.f._withCurrentView (http://localhost/4cc802552a1eca5cf1c8cd335aa43dfcf1501b7a.js?meteor_js_resource=true:59:23097) undefined
```

**Comment by whowillcare** [comment]
it's not just pasting image, it seems the latest changes to wekan had
removed summernote. I will investigate more.

**Comment by whowillcare** [comment]
@xet7 btw, I just found, after wekan being built, summernote package seemed to have been removed, for example, in snap, you cannot use richer text editor any more, because $().summernote is no longer a function. Did you notice that? thx

**Comment by xet7** [comment]
@whowillcare 

Thanks! I will add it back.

**Comment by xet7** [comment]
Please test newest Wekan v3.11 , does it work.

**Comment by xet7** [comment]
@whowillcare 

Thanks!

I did merge your pull request, but drag-drop and pasting attachments to Wekan attachments did not work for me, as shown in this issue first comment screenshot, I will test is later more because I don't know did I test it correctly. I don't know would it be possible to make drag-drop or paste directly to Wekan attachments area working.

Sometime there will be also feature to store attachments to some other storage #142 because it would be better for big files to not store on MongoDB, and then store only link to file. Maybe then also summernote attachments could be moved there by going through comments with code, moving base64 encoded files to external storage, and replacing those base64 encoded files with link to file. Well, that is, if you did not change those files to store directly to Wekan attachments, I did not check more closely.

**Comment by whowillcare** [comment]
just made a modification to store the attachments on server filesystem
instead of mongodb. the wekan backup script might need to be change as well

**Comment by xet7** [comment]
@whowillcare 

Would it be possible for you to add to Wekan Admin Panel some feature button that would make possible to move files from MongoDB to filesystem, and filesystem to MongoDB ?

**Comment by xet7** [comment]
@whowillcare 

Also, if attachments exist in both filesystem and MongoDB, and filesystem path is set, if file is not found from filesystem, could it be looked from database? So that attachments would not be lost?

**Comment by xet7** [comment]
@whowillcare 

Someone tested disabling summernote in Wekan Snap, and disabling did not work.

**Comment by xet7** [comment]
@whowillcare 

Can you add ATTACHMENTS_STORE_PATH also to these files?
- wekan/snap-src/bin/wekan-help
- wekan/Dockerfile
- wekan/docker-compose.yml
- wekan/start-wekan.sh
- wekan/start-wekan.bat
- wekan/releases/virtualbox/start-wekan.sh

**Comment by whowillcare** [comment]
Will do when I have some time tmr

**Comment by xet7** [comment]
Thanks!

Ctrl+V and drag to attachements popup was not fixed yet after https://github.com/wekan/wekan/pull/2593

**Comment by whowillcare** [comment]
I cannot reproduce this issue at all, I can ctrl+v picture in without any
issue. drag & drop might get disabled, because it messes up the system
sometime.

**Comment by yeyonghao1** [comment]
My system is CentOS 7.6.1810 with snap install wekan.

**Comment by nightsdark** [comment]
I have browser console reporting image sent in response after ctrl-v then box stays open and no response after hitting upload and image doesn’t appear in box. 

System Info Server: Ubuntu 16.04 running on VM 
 
   Wekan is Snap installed and nginx is set to allow (client_max_body_size 8M;)
   The system is working in all other ways including upload image from computer.                              

User System: Ubuntu 16.04  
                      Firefox 68.01 64bit Quantum

**Comment by yeyonghao1** [comment]
![image](https://user-images.githubusercontent.com/22569933/65369460-ab099500-dc7f-11e9-8088-3a5fb36d017f.png)
I still can't use it

**Comment by notz** [comment]
This issue should be fixed by #2754
